### DISCUSSION TITLE
qiita-cli, npx qiita publishする際にXに同時投稿する機能がほしい

### DISCUSSION THREAD

**Discussion by KOH6** [increments/qiita-discussions#543] [要望・アイデア・不具合]
[qiita-cli](https://github.com/increments/qiita-cli)リリースいただきありがとうございます。
早速使用しておりとても便利で感動しております。

画面側で投稿する際の設定と同様に、`npx qiita publish`する際にXに同時投稿する機能がほしいです。オプションで設定できる形でも問題ありません。
現在qiita-cliで投稿した後、手動で画面からXに投稿しているので少し手間がかかっています。

ご検討よろしくお願いいたします。

**Comment by getty104** [comment]
@KOH6 qiita-cliを使っていただきありがとうございます！
Qiitaとしてもより記事執筆しやすい体験を提供していきたいと考えておりますので、使い心地についてフィードバックをいただけて大変ありがたいです 🙏 
https://github.com/increments/qiita-discussions/discussions/542 と併せて検討させていただきます！

**Reply by KOH6** [reply]
承知いたしました！
はい！ぜひご検討いただけると嬉しいです！
